lib/rules/jsx-props-no-multi-spaces.js

```diff
@@ -40,12 +42,17 @@ module.exports = {
     function checkSpacing(prev, node) {
       const prevNodeEndLine = prev.loc.end.line;
       const currNodeStartLine = node.loc.start.line;
+      const nodeLineDifference = currNodeStartLine - prevNodeEndLine;
 
-      if (currNodeStartLine - prevNodeEndLine > 1) {
-        context.report({
-          node,
-          message: `Expected no line gap between “${getPropName(prev)}” and “${getPropName(node)}”`
-        });
+      if (nodeLineDifference > 1) {
+        for (let i = 0; i < nodeLineDifference - 1; i++) {
+          if (!sourceCodeArray[prevNodeEndLine + i].includes('//')) {
```

Comment:
there could also be comments like `/* foo */` or also like `<!-- bar -->` - can we add some tests for those? I have a feeling there's an eslint helper method here that can help ensure there's just whitespace and comments, rather than relying on introspecting text.